GLAA.Domain/Models/Licence.cs

```diff
@@ -52,12 +46,13 @@ public class Licence : IId, IAddressable
 
         public string CompaniesHouseNumber { get; set; }
         public DateTime? CompanyRegistrationDate { get; set; }
-        public bool? HasPAYEERNNumber { get; set; }
-        public string PAYEERNNumber { get; set; }
-        public DateTime? PAYEERNRegistrationDate { get; set; }
+        public bool? HasPAYENumber { get; set; }
+        public virtual ICollection<PAYENumber> PAYENumbers { get; set; } = new Collection<PAYENumber>();
```

Comment:
Never noticed you could do this before, nice!

Reply:
Yeah I think it's a 4.5 or 4.6 thing. Saves some of the `ctor` init stuff but I'm in two minds as it means for larger classes there are multiple places to check for setup code,